web/data/data.go

```diff
@@ -111,10 +113,19 @@ func createDoc(c echo.Context) error {
 		return err
 	}
 
+	if err := hooks.Run(instance, realtime.EventCreate, &doc, nil); err != nil {
+		return err
+	}
+
 	if err := permissions.Allow(c, permissions.POST, &doc); err != nil {
 		return err
 	}
 
+	errHoook := hooks.Run(instance, hooks.EventCreate, doc, nil)
```

Comment:
Shouldn't we call it after the creation has been successful? If there is a conflict, we should not call this, right?